src/mpyl/utilities/repo/__init__.py

```diff
@@ -4,7 +4,6 @@
 
 import json
 import logging
-import os
```

Comment:
Unnecessary import removed just to make sure we trigger another Dockerfile build.